docs/articles/documentation/extending-testcafe/index.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: docs
+title: Extending TestCafe
+permalink: /documentation/extending-testcafe/
+---
+# Extending TestCafe
+
+TestCafe is a powerful framework, however you may need a specific functionality that TestCafe does not provide initially.
```

Comment:
When the topic doesn't contain this sentence (it contains only info about the capability to extend TestCafe functionality by using plugins), Vasily ask me to write some intro.